src/lp.jl

```diff
@@ -0,0 +1,174 @@
+#=
+Interface to lrs_solve_lp
+
+=#
+
+
+mutable struct LRSLinprogSolution
+    status::Symbol  # :Optimal, :Infeasible, :Unbounded, :UserLimit, or :Error
+    objval::Union{Nothing,Rational{BigInt}}
+    sol::Vector{Rational{BigInt}}
+    attrs::Dict
+end
+
+
+function setobj(P::Ptr{Clrs_dic}, Q::Ptr{Clrs_dat},
+                c::Vector{<:Real}, maximize::Bool)
+    row = Vector{Rational{BigInt}}(undef, unsafe_load(P).d+1)
+    sgn = maximize ? 1 : -1
+    field = maximize ? :maximize : :minimize
+    row[1] = 0
+    row[2:end] = c * sgn
+    unsafe_field_store!(Q, field, Clrs_true)
+    setrow(P, Q, 0, row, true)
+end
+
+setobj(m::HMatrix, c::Vector{<:Real}, maximize::Bool) =
+    setobj(m.P, m.Q, c, maximize)
+
+@doc """
+    setobj(m::HMatrix, c::Vector{<:Real}, maximize::Boo)
+
+Set the objective function vector `c` for `m`. Set `maximize` to true (false,
+resp.) if the objective is to be maximized (minimized, resp.).
+""" setobj
```

Comment:
Doesn't it need to be right before `set_obj` ?

Reply:
It seems to work anywhere with `@doc`. Refer to [this](https://docs.julialang.org/en/v1/manual/documentation/#Advanced-Usage-1).